.travis.yml

```diff
@@ -8,8 +8,11 @@ node_js:
   - "iojs-v3"
   - "4"
   - "5"
+  - "6"
 env:
   - BROWSER=1
```

Comment:
I actually just removed the env switching entirely as the permutations were getting unwieldy in #17